packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx

```diff
@@ -128,26 +129,29 @@ const MedicationsDetailsTable: React.FC<ActiveMedicationsProps> = ({
             </p>
           </div>
           <p className={styles.bodyLong01}>
-            {medication.orderReasonNonCoded ? (
+            {medication.orderReasonNonCoded && (
               <span>
                 <span className={styles.label01}>{t('indication', 'Indication').toUpperCase()}</span>{' '}
                 {medication.orderReasonNonCoded}
               </span>
-            ) : null}
-            {medication.quantity ? (
+            )}
+            {medication.quantity && (
               <span>
                 <span className={styles.label01}> &mdash; {t('quantity', 'Quantity').toUpperCase()}</span>{' '}
                 {medication.quantity} {medication?.quantityUnits?.display}
               </span>
-            ) : null}
-            {medication.dateStopped ? (
+            )}
+            {(medication.dateStopped || medication.autoStopDate) && (
               <span>
                 <span className={styles.label01}>
-                  &mdash; {t('discontinuedDate', 'Discontinued date').toUpperCase()}
+                  &mdash; {t('discontinuedDate', 'Discontinued date').toUpperCase()}:{' '}
+                  {formatDate(new Date(medication.dateStopped || medication.autoStopDate))}
                 </span>
-                {formatDate(new Date(medication.dateStopped))}
+                <Tag type="gray" className={styles.tag}>
```

Comment:
Add some left margin to the tag so it has some breathing room from the rest of the content.